aten/src/ATen/native/quantized/library.cpp

```diff
@@ -187,6 +187,10 @@ TORCH_LIBRARY(quantized, m) {
   m.def(TORCH_SELECTIVE_SCHEMA("quantized::relu6(Tensor qx, bool inplace=False) -> Tensor"));
   m.def(TORCH_SELECTIVE_SCHEMA("quantized::leaky_relu(Tensor qx, Scalar negative_slope, bool inplace, float output_scale, int output_zero_point) -> Tensor"));
   m.def(TORCH_SELECTIVE_SCHEMA("quantized::sigmoid(Tensor qx, float output_scale, int output_zero_point) -> Tensor"));
+
+  // quantized ops implemented in cudnn, with QuantizedCUDA dispatch
+  // TODO: use the same signature as quantized::conv2d
+  m.def(TORCH_SELECTIVE_SCHEMA("quantized::conv2d_cudnn(Tensor act, Tensor weight, Tensor? bias, int[] stride, int[] padding, int[] dilation, int groups, float output_scale, int output_zero_point) -> Tensor"));
```

Comment:
I see that in native_functions.yaml there are other functions with `cudnn` in the name, I'm just wondering about the context about exposing the fact that cudnn is used versus hiding it in the implementation?

Reply:
we can hide it, we plan to merge this with the existing quantized::conv2d, but that's a bigger change, which involves things that is not relevant to cudnn, so want to land this one first and refactor later

Reply:
see the comment above the registration